From 6494f3f0b460d020122f3ac6910058cf0c1996ae Mon Sep 17 00:00:00 2001
From: Herman Chen <herman.chen@rock-chips.com>
Date: Mon, 8 Oct 2018 16:59:15 +0800
Subject: [PATCH] arm64: dts: rockchip: rename vpu to vpu_service

/dev/vpu is the old device name. New driver always use vpu_service
as default vcodec driver name.

Change-Id: I19896d92336291d0035fa4ed7b466cb64fd96d78
Signed-off-by: Herman Chen <herman.chen@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 00eb20c03c3d..099c00c03d71 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -1024,7 +1024,7 @@
 		status = "disabled";
 	};
 
-	vpu: vpu@ffb80000 {
+	vpu_service: vpu_service@ffb80000 {
 		compatible = "rockchip,vpu_service";
 		reg = <0x0 0xffb80000 0x0 0x800>;
 		interrupts = <GIC_SPI 93 IRQ_TYPE_LEVEL_HIGH>,
-- 
2.35.3

